crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -477,32 +412,14 @@ impl CheckpointExecutorEventLoop {
         Ok(())
     }
 
-    async fn reconfig(
+    fn create_committee_object(
```

Comment:
I believe you don't need this function. Can simply call the constructor of `Committee`

Reply:
I'm creating the `Committee` in two different places. This is just a helper to clean up the constructor call, which requires some type conversion